customizations/high_availability/standard_with_upgrades/README.md

```diff
@@ -0,0 +1,31 @@
+## High Availability configuration: Standard with upgrades
+
+### Requirements
+
+- NodePool with at least 2 nodes
```

Comment:
NodePool with at least 2 nodes and with a total of 33gb RAM and 20 CPUs for workloads in order to avoid evictions if the minimal number of replicas approach to the limits.

Reply:
The **TOTAL** column is calculated against the `maxReplicas` figure, therefore the resources required for the minimal number of replicas are `26gb RAM and 15 CPUs`, since the minReplicas is 2.